packages/cli/test/integration/cli/cli.integration.js

```diff
@@ -49,7 +49,10 @@ describe('cli', () => {
     expect(entries).to.not.containEql('Available commands:');
   });
 
-  it('saves command metadata to .yo-rc.json', () => {
+  it('saves command metadata to .yo-rc.json', function () {
+    // This test can be slow under parallel mode
+    // eslint-disable-next-line @typescript-eslint/no-invalid-this
```

Comment:
This is not ideal. I opened a PR to avoid `eslint-disable-next-line`, see https://github.com/strongloop/loopback-next/pull/5925